### ISSUE TITLE
Python implement for Example 5.5 does not have successful example

### ISSUE THREAD

**Issue by qcmiao1998** [samaritanhu/Introduction-to-Algorithms#12]
@samaritanhu [Python 5.5](https://github.com/samaritanhu/Introduction-to-Algorithms/blob/master/codes/python/Chapter%205/Example%205.5.ipynb)

**Comment by samaritanhu** [comment]
Yes, you pointed out a great problem... I would figure it out. And by the way, Python's array should not have been written like `P[i, j]`, my fault.

**Comment by qcmiao1998** [comment]
If you need, some test examples for this problem can be found in [my repo](https://github.com/qcmiao1998/Introduction-to-Algorithms-Csharp-Codes/blob/d3b331ce88046c6991f2604e049a19389a18eef7/Test/Chapter5.cs#L26).
